CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### Improvements
 - feat: enable use of replica database (delegating the choice to `DATABASES_ROUTER`) ([#359](https://github.com/jazzband/django-auditlog/pull/359))
 - Add `mask_fields` argument in `register` to mask sensitive information when logging ([#3710](https://github.com/jazzband/django-auditlog/pull/310))
+- Django: Drop 2.2 support
```

Comment:
Maybe precise that `django_jsonfield_backport` is not required anymore ?

Reply:
Done!